configmap/Tiltfile

```diff
@@ -0,0 +1,63 @@
+# -*- mode: Python -*-
+
+def configmap_yaml(name, namespace="", from_file=None, watch=True):
+  """Returns YAML for a generic configmap
+
+  Args:
+    name: The configmap name.
+    namespace: The namespace.
+    from_file: Use the from-file configmap generator. May be a string or a list of strings.
+       Example: ["grafana.ini=path/to/grafana.ini"]
+    watch: Reruns the Tiltfile and re-deploys automatically if the from-files change.
+       Defaults to true.
+
+  Returns:
+    The configmap YAML as a blob
+  """
+
+  args = [
+    "kubectl",
+    "create",
+    "configmap",
+    name,
+  ]
+
+  if namespace:
+    args.extend(["-n", namespace])
+
+  generator = False
+  if from_file:
+    if type(from_file) == "string":
```

Comment:
ffti, but a pattern I've liked for this:

```
if type(from_file) == "string":
  from_file = [from_file]
if type(from_file) == "list":
  # this block now handles both cases
else:
  # error
```

Reply:
i like that! done